optaweb-vehicle-routing-docs/src/main/asciidoc/index.adoc

```diff
@@ -1 +1,47 @@
-= OptaWeb Vehicle Routing User Guide
+[id="vrp-quick-start"]
+
+= Using OptaWeb Vehicle Routing
+
+:context: vrp-user-guide
+
+As a developer, you can use OptaWeb Vehicle Routing to optimize your vehicle fleet deliveries.  In this guide, you will create and run a sample OptaWeb Vehicle Routing application.
```

Comment:
I suggest to follow the [one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) code style. I hope this doesn't conflict with the docs team code style.

Reply:
You mean in text, rather than in code examples? In actual text, we group sentences in paragraphs. Did I misunderstand?

Reply:
Yes, there seem to be a bit of misunderstanding. Let me clear it up.

I'm talking about plain text (in paragraphs). The rule I referenced requires that each sentence starts on a new line. This rule aims at paragraph with multiple sentences. When following the rule, each Asciidoc source line contains only one sentence. Note that a line break between sentence won't terminate the paragraph so sentences will be still grouped (paragraph is terminated by a *blank line*).

See the link in previous comment for the benefits of this rule.

For example this text (above the comments) has two sentences that are both on line 7. My suggestion is to place a line break between them so the result would look like:
```diff
As a developer, you can use OptaWeb Vehicle Routing to optimize your vehicle fleet deliveries.
In this guide, you will create and run a sample OptaWeb Vehicle Routing application.
```